src/coreclr/tools/Common/Compiler/TypeExtensions.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System;
+using System.Runtime.CompilerServices;
```

Comment:
Is this `using` really needed?

Reply:
Yup, Unsafe.NullRef so that users who don't need static method support av if they accidentally pass static methods.